.github/workflows/maven.yml

```diff
@@ -93,7 +93,7 @@ jobs:
       - name: DataDog sourcemap upload
         # only on push to main!
         if: github.repository == 'unicode-org/cldr' && github.event_name == 'push' && github.ref == 'refs/heads/main' && github.event.inputs.git-ref == ''
-        run: npx --package=@datadog/datadog-ci datadog-ci sourcemaps upload tools/cldr-apps/src/main/webapp/dist/ --minified-path-prefix=/cldr-apps/dist/ --release-version=r${{ github.event.inputs.git-ref }} --service=surveytool
```

Comment:
`git-ref` is only applicable for manual pushes (such as we use for production or staging), not for the automated push to smoketest.